Documentation/Security/GuidelinesAdministrators/RestrictAccessToFiles.rst

```diff
@@ -8,6 +8,10 @@
 Restrict access to files on a server-level
 ==========================================
 
+.. hint::
+
+   TYPO3 provides a recommended :file:`.htacess` file. See :ref:`t3install:system-environment` .
```

Comment:
This should be moved to the *Apache and Microsoft IIS web servers* section below.